### PR TITLE
[Feature] Hide biometrical if device does not support it

### PR DESCRIPTION
**Pull request by JuancaG05**
## Related Issues
App: https://github.com/owncloud/android/issues/3217

- [x] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

## QA

QA checks: https://github.com/owncloud/android/pull/3230#issuecomment-838242849

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsSecurityFragment.kt [hunk 1]
```diff
@@ -155,25 +155,23 @@ class SettingsSecurityFragment : PreferenceFragmentCompat() {
...
-            }
-            prefBiometric?.setOnPreferenceChangeListener { preference: Preference?, newValue: Any ->
-                val incomingValue = newValue as Boolean
-
-                // Biometric not supported
-                if (incomingValue && biometricManager?.isHardwareDetected == false) {
-                    showMessageInSnackbar(getString(R.string.biometric_not_hardware_detected))
-                    return@setOnPreferenceChangeListener false
+            if (biometricManager?.isHardwareDetected == false) { // Biometric not supported
+                screenSecurity?.removePreference(prefBiometric)
```
Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsSecurityFragment.kt:159]: Reformat this file 👍

**Review by abelgardep — CHANGES_REQUESTED** [review]
Some tiny suggestions here @JuancaG05

File: owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsSecurityFragment.kt (hunk 1 above)
Comment (JuancaG05) [reply owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsSecurityFragment.kt:159]: It is already formatted with Android Studio guidelines! No changes if I click on "Reformat code" 😕

Comment (abelgardep) [reply owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsSecurityFragment.kt:159]: My bad :(

**Review by abelgardep — APPROVED** [review]
Approved! Ready to QA @jesmrec

**Comment by jesmrec** [comment]
QA checks:

- [X] Device with no biometrical support - Passcode enabled, Biometrical hidden
- [X] Device with no biometrical support - Pattern enabled, Biometrical hidden
- [X] Device with biometrical support - Passcode and pattern disabled, Biometrical displayed and disabled
- [X] Device with biometrical support - Passcode enabled, Biometrical displayed and enabled
- [X] Device with biometrical support - Pattern enabled, Biometrical displayed and enabled

**Comment by jesmrec** [comment]
Approved

**Pull request merged**